include/deal.II/sundials/kinsol.h

```diff
@@ -214,6 +214,39 @@ namespace SUNDIALS
         picard = KIN_PICARD,
       };
 
+
+      /**
+       * Orthogonalization strategy used for fixed-point iteration with
+       * Anderson acceleration. While `modified_gram_schmidt` is a good default
+       * choice, it suffers from excessive communication for a growing number of
+       * accleration vectors. Instead, the user may use one of the other
+       * strategies. For a detailed documentation consult the SUNDIALS manual.
+       */
+      enum Orthogonalization
```

Comment:
This has similarities with, and two members of the same name, in https://github.com/dealii/dealii/blob/d258e5169a4cb2cbbd3c2facc60f82c798216939/include/deal.II/lac/orthogonalization.h#L27-L44
While the two schemes are not the same, I like that we have the same enum names. I would prefer if we also call this enum `OrthogonalizationStrategy` and like to this other class, because users might be confused if we expose different structs (albeit clear to me at this point, but users might be less experienced than we are).